python/hail/utils/tests.py

```diff
@@ -59,27 +59,26 @@ def test_hadoop_methods(self):
 
         self.assertEqual(b, b2)
 
-    def test_queue(self):
-        q = Queue()
-        self.assertEqual(list(q), [])
-        if q:
-            self.fail('empty queue had an implicit boolean value of True')
+    def test_linked_list(self):
+        ll = LinkedList(int)
+        self.assertEqual(list(ll), [])
+        if ll:
+            self.fail('empty linked list had an implicit boolean value of True')
 
-        q = q.push(5).push(2)
+        ll2 = ll.push(5).push(2)
 
-        self.assertEqual(list(q), [5, 2])
-        self.assertEqual(len(q), 2)
```

Comment:
did you want a `LinkedList.__len__` method? Slightly more expensive unless you store the length in the node, idk if it's useful

Reply:
don't really need to use it anywhere, so parsimony says no.

Reply:
👍